wgpu/Cargo.toml

```diff
@@ -34,6 +34,7 @@ raw-window-handle = "0.3"
 log = "0.4"
 guillotiere = "0.6"
 futures = "0.3"
+kamadak-exif = "0.5"  # TODO(akavel): cfg(only when jpeg enabled) ?
```

Comment:
Actually, since writing this TODO I realized that the library [seems to work for at least TIFF, JPEG, HEIF, PNG, and WebP](https://docs.rs/kamadak-exif/0.5.4/exif/index.html), so I planned to remove the comment before submitting, but missed it. Leaving the decision as to whether to clean it to you now then.

Reply:
Yes. Since it's a small dependency, we can just add it unconditionally for simplicity.